benchmarks/targets/transpile-brs.js

```diff
@@ -20,7 +20,7 @@ module.exports = async (suite, name, brighterscript, projectPath, options) => {
     }
 
     if (files.length === 0) {
-        console.log('[transpile-brs] No brs|bs|d.bs files found in program');
+        console.log('[allow-brighterscript-in-brightscript] No brs|bs|d.bs files found in program');
```

Comment:
I don't think this one needs changed. The name of this benchmark is "transpile-brs", and is unrelated to this new feature.